OasisDEX/oasis-borrow
# 🧿 Ajna common context for open and manage position

# [🧿 Ajna common context for open and manage position](https://app.shortcut.com/oazo-apps/story/7676/manage-borrow-position-ui-components)

POC of `AjnaProductController` component that could be used for both open and manage positions pages.

![image](https://user-images.githubusercontent.com/16230404/215520597-e920fc51-65f2-4e99-a385-2d3a2546e093.png)
  
## Changes 👷‍♀️

- Changed open flow url from `/ajna/[product]/[pair]` to `/ajna/open/[product]/[pair]`,
- created `AjnaProductController` component that handles open and manage borrow positions flows,
- created empty component for form manage step.
  
## How to test 🧪

See if both variants are working:
- `/ajna/open/borrow/ETH-DAI` - example pair, should also work with couple others, see `/features/ajna/common/consts.ts`,
- `/ajna/position/2137` - or any other number or string, it's not validated for now.
